mbuild/tests/test_hoomd.py

```diff
@@ -254,3 +254,13 @@ def test_auto_scale_forcefield(self, ethane):
         # The first element is empty, the next element should be ['opls_135', '1.0000', '1.0000']
         assert pair_coeffs[1][1] == "1.0000"
         assert pair_coeffs[1][2] == "1.0000"
+
+    def test_hoomdsimulation_nlist(self, ethane):
```

Comment:
I think this test should go above in the other test class in this file. Its begins [on line 12](https://github.com/mosdef-hub/mbuild/pull/913/files#diff-24f4b0d616ce2c38c52d22707d2333fe4940cafa5f104033c31781d19592d866L12)


Those tests ensure that hoomd is installed to run the tests, while this test class is focused on the hoomdxml writer (technically independent if hoomd is installed or not). After that migration, this looks good to go!

Reply:
Oh, good catch! I thought that was the test class I was putting it in--oops 🤦‍♀️